lib/tainbox/instance_methods.rb

```diff
@@ -16,7 +16,7 @@ def initialize(*args)
 
   def attributes
     self.class.tainbox_attributes.map do |attribute|
-      [attribute, send(attribute)] if respond_to?(attribute, true)
+      [attribute, send(attribute)] if respond_to?(attribute)
```

Comment:
Private reader should not be exposed. Also you are breaking back-comp here

Reply:
>Private reader should not be exposed
He is actually _fixing_ this. Now only public readers will show up in `attributes`

Reply:
Probably it's better to compare `#attributes` behavior.
This code has similar results on master branch and on the pull request branch:
```
Class.new do
  include Tainbox
  attribute :a1
  attribute :a2, reader: false
  attribute :a3, writer: false
end.new.attributes
# => {:a1=>nil, :a3=>nil}
```
For me it's not really clear why `respond_to?` included private methods in its result before.
So let me know if I miss something.